package.yaml

```diff
@@ -1,23 +1,16 @@
-name: haskell-library-template
-version: 0.0.0.0
+name: aeson-compat
+version: 0.0.0.1
 maintainer: Freckle Education
 category: Utils
-github: freckle/haskell-library-template
-synopsis: Short synopsis here
+github: freckle/aeson-compat
+synopsis: Compatibility modules between major aeson versions
 description: |
-  Longer description here
-
-  Haddocks are more valuable if you reproduce the main README example here, but
-  you can also just direct users to the README if you prefer not to take on that
-  duplication burden.
+  TODO
```

Comment:
Synopsis is longer than description; I don't think Hackage will take this?

Reply:
I'll fix it one way or another next substantive change we make.